eve/main.yml

```diff
@@ -30,3 +41,18 @@ stages:
         urls:
           - "*.iso"
           - "SHA256SUM"
+
+  lint:
+    worker:
+      type: docker
```

Comment:
tip: try not to use worker of type `docker` anymore! and only use `kube_pod`

Reply:
Yes please, you have sample here that were already do some linting: https://github.com/scality/metalk8s/blob/development/1.3/eve/main.yml#L180-L193

Reply:
Well actually there's something I'm not understanding. On one side you're setting your worker of type `docker` on the other side you have a `pod` description file which is meant for worker of type `kube_pod`.

I'm confused

Reply:
It was originally a pod, then I took into account a comment while unaware that `kube-pod` is the way to do it, and I haven't committed the new directory structure + file deletion. Reverting to a pod will be quick :)

Reply:
okkk good good!

Reply:
> tip: try not to use worker of type `docker` anymore! and only use `kube_pod`

Yup my bad, I was the one that requested the change (it felt unnecessary to have access to Kube for a simple linting step). Maybe you could add a **[deprecated]** mention in Eve docs for that builder @tcarmet @Zempashi (in case other developers miss that info about preferring `kube_pod`)?

Reply:
very good point @gdemonet ! [Here you go](https://scality.atlassian.net/browse/EVE-1036)